src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -250,3 +269,150 @@ def snapshot(self) -> list[ClusterInfo]:
     def _try_fetch(self) -> list[ClusterInfo]:
         for row in self._fetch(f"SELECT * FROM {self._schema}.{self._table}"):
             yield JobInfo(*row)
+
+
+class ExternallyOrchestratedJobTaskCrawler(CrawlerBase):
+    def __init__(self, ws: WorkspaceClient, sbe: SqlBackend, schema):
+        super().__init__(sbe, "hive_metastore", schema, "job_runs", ExternallyOrchestratedJobTask)
```

Comment:
`job_runs` may be too generic here for a table name

Reply:
will rename it